jvcproj/plugin.yaml

```diff
@@ -0,0 +1,25 @@
+# Metadata for the Smart-Plugin
+plugin:
+    # Global plugin attributes
+    type: interface                 # plugin type (gateway, interface, protocol, system, web)
+    description:
+        de: 'Plugin um JVC D-ILA Projektoren zu kontrollieren und durch jvcprojectortools erstellte Gammakurven zu uebertragen'
+        en: 'plugin to control JVC D-ILA projectors and transfer gammatables generated with jvcprojectortools'
+    maintainer: NPortz
+    tester: 
+#    keywords: iot xyz
+#    documentation: https://github.com/smarthomeNG/smarthome/wiki/CLI-Plugin        # url of documentation (wiki) page
+#    support: https://knx-user-forum.de/forum/supportforen/smarthome-py
+
+    version: 1.0.0                 # Plugin version
+    sh_minversion: 1.1             # minimum shNG version to use this plugin
+#    sh_maxversion:                 # maximum shNG version to use this plugin (leave empty if latest)
+    multi_instance: False          # plugin supports multi instance
+    classname: JVC_DILA_Control    # class containing the plugin
+
+parameters:
+    # Definition of parameters to be configured in etc/plugin.yaml
```

Comment:
If you document the plugin parameters here you could benefit from some default values also declare here instead of checking in the code. Take a look into the [docs](http://smarthomeng.de/developer/development_plugin/plugin_metadata.html) to see how it works.

Reply:
Yes, i will do that!